docs/source/guides/pipelines.mdx

```diff
@@ -0,0 +1,126 @@
+<!---
+Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Inference pipelines with AWS Neuron (Inf2/Trn1)
+
+The [`~pipelines.pipeline`] function makes it simple to use models from the [Model Hub](https://huggingface.co/models)
+for accelerated inference on a variety of tasks such as text classification, question answering and image classification.
+
+<Tip>
+
+You can also use the
+[pipeline()](https://huggingface.co/docs/transformers/main/en/main_classes/pipelines#pipelines) function from
+Transformers and provide your Optimum model class.
+
+</Tip>
+
+Currently the supported tasks are:
+
+* `feature-extraction`
+* `fill-mask`
+* `text-classification`
+* `token-classification`
+* `question-answering`
+* `zero-shot-classification`
+
+## Optimum pipeline usage
+
+While each task has an associated pipeline class, it is simpler to use the general [`~pipelines.pipeline`] function which wraps all the task-specific pipelines in one object.
+The [`~pipelines.pipeline`] function automatically loads a default model and tokenizer/feature-extractor capable of performing inference for your task.
+
+1. Start by creating a pipeline by specifying an inference task:
+
+```python
+>>> from optimum.neuron.pipelines import pipeline
+
+>>> classifier = pipeline(task="text-classification")
+```
+
+2. Pass your input text/image to the [`~pipelines.pipeline`] function:
+
+```python
+>>> classifier("I like you. I love you.")  # doctest: +IGNORE_RESULT
+[{'label': 'POSITIVE', 'score': 0.9998838901519775}]
+```
+
+_Note: The default models used in the [`~pipelines.pipeline`] function are not optimized for inference or quantized, so there won't be a performance improvement compared to their PyTorch counterparts._
+
+### Using vanilla Transformers model and converting to AWS Neuron
```

Comment:
I think we should also push users to use the CLI to do it. Once it is done they do not have to re-run the export and can use the exported file instead, which I think it what you want in a production setting.